CHANGELOG.md

```diff
@@ -1,11 +1,56 @@
 ## CHANGELOG
 
+### `v1.31`
+
+A big release after a long time dormant!
+
+#### Notable changes
+* Use Elixir Logger as the default agent logger [#455](https://github.com/newrelic/elixir_agent/pull/455)
+  - Changes where internal New Relic agent logs will go by default
+* Use logger primary filter to hook into error reporting [#466](https://github.com/newrelic/elixir_agent/pull/466)
+  - No longer need to add `ErrorLogger` to your `:logger` `:backend` config
+* Finch Instrumentation [#469](https://github.com/newrelic/elixir_agent/pull/469)
+  - No longer need to `@trace` with `category: :external` to instrument Finch based HTTP requests
+
+#### Features
+* Make Distributed Tracing configurable [#457](https://github.com/newrelic/elixir_agent/pull/457)
+* Support ignoring web transaction paths via configuration [#451](https://github.com/newrelic/elixir_agent/pull/451)
+* Support host display name [#464](https://github.com/newrelic/elixir_agent/pull/464)
+* Oban Instrumentation [#463](https://github.com/newrelic/elixir_agent/pull/463)
+* Absinthe Instrumentation [#471](https://github.com/newrelic/elixir_agent/pull/471)
+* Added `NewRelic.notice_error` to report rescued exceptions [#470](https://github.com/newrelic/elixir_agent/pull/470)
+* Added `NewRelic.add_span_attributes` [#478](https://github.com/newrelic/elixir_agent/pull/478)
```

Comment:
Also looks like a macro for spans was added?

Reply:
Looks like this is a nice new feature:

```
NewRelic.span("do.some_work", user_id: "abc123") do
  # do some work
end
```

Reply:
Added that to the changelog and went through and updated a lot of documentation